backend/app.py

```
@@ -23,7 +23,7 @@ def login():
     hashed = hashlib.md5(password.encode("utf-8")).hexdigest()
```

Ruff (0.13.3)

Local variable `hashed` is assigned to but never used

Remove assignment to unused variable `hashed`

(F841)